backend/games/word_guesser/WordGuesser.py

```diff
@@ -0,0 +1,124 @@
+from enum import IntEnum
+from random import randint
+from typing import Dict, List
+
+MAX_GUESSES = 6
+
+# initialize valid guesses and possible solutions
+SOLUTIONS: str
+VALID_WORDS: str
+
+try:
+    solutions_file = open(
+        "backend/games/word_guesser/WordGuesser_Solutions.txt"
+    )
+    guesses_file = open(
+        "backend/games/word_guesser/WordGuesser_ValidGuesses.txt"
+    )
+    SOLUTIONS = solutions_file.read()
+    SOLUTION_LIST = SOLUTIONS.splitlines()
+
+    VALID_WORDS = guesses_file.read()
+    VALID_WORDS_LIST = VALID_WORDS.splitlines()
+except:
+    print("Files not readable. Check file or file path.")
+
+
+class CorrectnessScore(IntEnum):
+    correct = 3
+    in_word = 2
+    not_in_word = 1
```

Comment:
This enum should still use string values, because the rest of the app is expecting to get the strings we had before.

Also, usually the enum values are all caps.